Player.gd

```diff
@@ -93,17 +93,12 @@ func calculate_velocity(delta):
 
 	if fire:
 		var instance = Bullet.instance()
-		add_child(instance)
-		instance.look_at(get_global_mouse_position())
-			
-		if direction == RIGHT:
-			instance.linear_velocity = Vector2(1000, 0).rotated(instance.rotation)
-			instance.damage = bullet_damage		
-			instance.connect("kill_obtained", self, "on_kill")
-		else:
-			instance.linear_velocity = Vector2(-1000, 0).rotated(-instance.rotation)
-			instance.damage = bullet_damage		
-			instance.connect("kill_obtained", self, "on_kill")
+		get_parent().add_child(instance)
+		instance.position = $RegularFirePoint.global_position
+		instance.look_at(get_global_mouse_position())	
+		instance.linear_velocity = Vector2(1000, 0).rotated(instance.rotation)
```

Comment:
We should extract the bullet speed to a constant in case we want to change it later.